src/serac/physics/utilities/functional/functional.hpp

```diff
@@ -522,6 +561,20 @@ class Functional<test(trial)> : public mfem::Operator {
    * @brief The gradient object used to implement @p GetGradient
    */
   mutable Gradient grad_;
+
+  /**
+   * @brief The element tangent matrix vector for ComputeElementMatrices() and UpdateAssembledSparseMatrix() calls
+   */
+  mutable mfem::Vector K_e_;
+
+  /**
+   * @brief Local AssembledSparseMatrix storage for ComputeElementMatrices
+   *
+   * If unique_ptr is empty, construct AssembledSparsMatrix.
+   *
+   * @return reference to internal AssembledSparseMatrix
```

Comment:
it looks like `assembled_spmat_` is a member variable, but the documentation makes it sound like a function (having a return value)